.travis.yml

```diff
@@ -36,9 +38,9 @@ script:
 branches:
   only:
   - master
-  - /^(dawn)\/.*$/
-  - /^(runtime)\/.*$/
-  - /^(test)\/.*$/
+  - /^dawn\/.*$/
+  - /^runtime\/.*$/
+  - /^test\/.*$/
 deploy:
```

Comment:
Ugh, of course I'd miss something. I'll back out those changes since there doesn't seem to be an easy way to resolve them.